lib/iris/tests/test_basic_maths.py

```diff
@@ -589,9 +589,9 @@ def test_apply_ufunc(self):
             in_place=False,
         )
 
-        ans = a.data**2
+        answer = a.data**2
```

Comment:
Wait - can we only use proper words as variable names now?

Reply:
Nope, we can easily add exceptions... the spell-checker obviously tried to correct this to something like `ans->and` (and a bunch of other suggestions).

To be fair `ans` is kinda weak as a variable name. Something like `expected` might be better? Dunno.

Anyways, we could add `ans` as a legitimate word to ignore, but then we run the risk of missing a genuine typo when the author intended `and` and put `ans` instead.

I'm not fussed really, happy to go with whatever you suggest/recommend.